pkg/kubecost/asset.go

```diff
@@ -2931,7 +2927,8 @@ func (as *AssetSet) accumulate(that *AssetSet) (*AssetSet, error) {
 
 type AssetSetRange struct {
 	sync.RWMutex
-	assets []*AssetSet
+	assets    []*AssetSet
+	FromStore string
```

Comment:
Same thing about a comment on this field.

Reply:
Done